sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -32,7 +32,6 @@ export const ServiceAuth = {
  */
 export const ServiceEnvironmentVariable = {
   PLAYWRIGHT_SERVICE_OS: "PLAYWRIGHT_SERVICE_OS",
-  PLAYWRIGHT_SERVICE_RUN_ID: "PLAYWRIGHT_SERVICE_RUN_ID",
```

Comment:
You need to remove this reference from the getDefaultRunId method
https://github.com/Azure/azure-sdk-for-js/blob/d34dc4e76ca179131c6e22e8c1ed0885a616378d/sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts#L67